src/quote.js

```diff
@@ -93,6 +93,20 @@ class Quote {
                           .then(setPaymentMediums);
     }
   }
+
+  getSellPaymentMediums () {
```

Comment:
This method name is misleading. It should really be `getSellPaymentAccounts`, but I don't think that's the right approach. Gettting and adding bank accounts should be an instance method on the `PaymentMedium` class.

PaymentMedium: the kind of thing used to pay, e.g. bank or card. This list of options is returned by `getPaymentMediums()`.

PaymentAccount: e.g. one bank account, or one creditcard (if we had such fine-grained control, but we don't, since that's all handled in the ISX frame)

I don't think this PR is needed. The consumer of this class, should just call `getPaymentMethods()` on the quote object, and `Quote` should should do the right thing, regardless of whether it's buy or sell. In the case of sell, it should only return bank. So something like this should (be made to) work:

```js
quote.getPaymentMediums().then((mediums) => {
  mediums.bank.getPaymentAccounts().then(bankAccounts => {
    bankAccounts[0].sell(); // -> trade
})
});

bankdetails = {....}
quote.getPaymentMediums().then((mediums) => {
  mediums.bank.addPaymentAccount(bankdetails);
})
});
```